include.xml

```diff
@@ -2,11 +2,12 @@
 <project>
 	<icon path="assets/images/logo/HaxeFlixel.svg" />
 	
-	<assets path="assets/sounds" include="*.ogg" if="html5" unless="unit-test" />
+	<assets path="assets/sounds" include="*.m4a" if="html5" unless="unit-test" />
 	<assets path="assets/sounds" include="*.mp3" if="flash" embed="true" />
-	<assets path="assets/sounds" include="*.ogg" unless="flash || html5" embed="true" />
+	<assets path="assets/sounds" include="*.ogg" if="native" embed="true" />
+	
 	<assets path="assets/fonts/nokiafc22.ttf" embed="true" />
-	<assets path="assets/fonts/arial.ttf" embed="true" unless="FLX_NO_DEBUG" />
+	<assets path="assets/fonts/montserrat-Regular.ttf" embed="true" if="debug" />
```

Comment:
Btw, the `unless="FLX_NO_DEBUG"` wasn't working because it's generally defined after flixel is included, and order matters in xml config files. I don't think there's a way to do this reliably there, which is why I preferred the `@:font` embedding - which is still broken on Android to my knowledge. :(